app/views/reviews/index.html.slim

```diff
@@ -2,45 +2,97 @@
 - content_for :page_heading, "My Reviews"
 
 - if @reviews.size > 0
+  .row
+    .col-xs-6.myreviews-stats-col1
+      .container-fluid#total-reviews-container
```

Comment:
Since we don't want to use ids for our css, you'll need to go through here as well and change them to regular classes.